tensorboard/webapp/runs/views/runs_table/runs_table_container.ts

```diff
@@ -816,6 +817,11 @@ export class RunsTableContainer implements OnInit, OnDestroy {
   orderColumns(newHeaderOrder: ColumnHeader[]) {
     this.store.dispatch(runsTableHeaderOrderChanged({newHeaderOrder}));
   }
+
+  useDataTable() {
+    console.log('forceLegacyTable', this.forceLegacyTable);
+    return this.HParamsEnabled.value && !this.forceLegacyTable;
```

Comment:
Can you rename HParamsEnabled to hparamsEnabled or hParamsEnabled?

Reply:
Changed.